pyproject.toml

```diff
@@ -26,6 +26,7 @@ dependencies = [
   "deprecated>=1.2.14",
   "copier",
   "pytest",
+  "plumbum",
```

Comment:
I'm globally reluctant to introduce a new dependecy that I never use and I won't be able to maintain, let's try to solve the 3.8 git identity error first but I would like to see how to do the same without the "git" method trick which is in the end only changing "cwd" and introducing 4 environment variables that you could set as parameters of the command itself (which I feel safer in this case)

Reply:
Sure, let me try to solve that first to get a passing PR.